htdocs/core/lib/functions.lib.php

```diff
@@ -8771,7 +8771,19 @@ function complete_head_from_modules($conf, $langs, $object, &$head, &$h, $type,
 							complete_substitutions_array($substitutionarray, $langs, $object, array('needforkey'=>$values[2]));
 							$label = make_substitutions($reg[1], $substitutionarray);
 						} else {
-							$label = $langs->trans($values[2]);
+							$labeltemp = explode(',', $values[2]);
+							if (is_array($labeltemp)) {
```

Comment:
labeltemp is result of explode so it will always be an array ?  (even if there is one element)

Reply:
Yes i enderstand 'if (is_array($labeltemp)) {' is always true i change

Reply:
Done